### ISSUE TITLE
Improve error / readability of Verify-Links.ps1

### ISSUE THREAD

**Issue by jspaith** [Azure/azure-sdk-for-c#1800]
First I *love* Verify-Links.ps1 and anything that automatically catches errors.

That said, the errors it's generated on a recent checkin (#1787) were hard to follow for me.  At issue is the readme.md is pointing to non-existant .c files which is a good catch

```
Found 97 links on page file:///D:/a/_work/1/s/sdk/samples/iot/README.md
##[warning][404] broken link ***/blob/master/sdk/samples/iot/paho_iot_pnp_sample.c
##[warning][404] broken link ***/blob/master/sdk/samples/iot/paho_iot_pnp_with_provisioning_sample.c
##[warning][404] broken link ***/blob/master/sdk/samples/iot/paho_iot_pnp_component_sample.c
```

The issue is I thought that the issue was in the files themselves and links they have, and not the readme.  It'd make it easier if maybe:
* There could be something to more clearly tie the warning to the error in the readme.md.  I thought the "found 97" meant success.  So like putting in "/blah/blah/iot/readmemd: broken link .../pnp_sample.c..
* The ***/blob/... also threw me because I thought it was pointing at the file itself having broken links.  Could we have the error instead have the actual URL as is in the file, so the fully qualified http://.../pnp_sample.c so this pops more?

**Comment by weshaggard** [comment]
Thanks for the feedback. 

The `***/` is a side-effect Devops treating the string https://githb.com/Azure/azure-sdk-for-c as a secret. I'll see if I can figure out what is marking that as a secret

Instead of "Found 97 links" would it be better if it said something like `Checking 97 links found on page file:///D:/a/_work/1/s/sdk/samples/iot/README.md`? 

Also did you see the summary at the bottom of the log? We try to group all the failures together there as well.

**Comment by jspaith** [comment]
Ahh - I didn't see the summary at the bottom, I think I just zeroed in on the errors!  But yes that would've solved this problem for me.

>Instead of "Found 97 links" would it be better if it said something like Checking 97 links found on page file:///D:/a/_work/1/s/sdk/samples/iot/README.md?

I do; I think I would've mentally mapped this up a bit clearer with that.
